includes/models/model.llms.course.php

```diff
@@ -218,11 +218,12 @@ public function get_categories( $args = array() ) {
 	 *
 	 * @since 1.0.0
 	 * @since 3.24.0 Unknown.
+	 * @since [version] Added support for showing multiple difficulties.
 	 *
 	 * @param string $field Optional. Which field to return from the available term fields.
 	 *                      Any public variables from a WP_Term object are acceptable: term_id, name, slug, and more.
 	 *                      Default is 'name'.
-	 * @return string
+	 * @return string|array
```

Comment:
as discussed, since we don't return an array, there's no reason to modify this, we always return a string.
```suggestion
	 * @return string
```